### PR TITLE
EDU-704 - Rework favorites tab UI to inline with rooms tab and user page

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-704
Base on https://github.com/educandu/educandu/pull/533, which has to be merged first.

Screen Shot 2022-07-29 at 12 54 15

### CODE REVIEW

File: src/components/favorites-tab.yml [hunk 1]
```diff
@@ -11,5 +11,14 @@ targetDeletedMessage:
   en: The target of this link has been deleted
   de: Das Ziel dieses Links wurde gelöscht
 info:
-  en: Here you can find all documents, rooms and users that you have marked as favorite.
-  de: Hier finden Sie alle Dokumente, Räume und Benutzer, die Sie als Favorit markiert haben.
+  en: Here you can find all users, rooms and documents that you have marked as favorite.
+  de: Hier finden Sie alle Benutzer, Räume und Dokumente, die Sie als Favorit markiert haben.
+favoriteUsers:
+  en: Favorite users
+  de: Favorite Benutzer
```
Comment (ahelmberger) [inline src/components/favorites-tab.yml:18]: "Benutzer, die Sie als Favorit hinzugefügt haben" (probably than also align English "Users you have added to your favorites". Also I realized the english "add **to** favorites" would be also better in German "**zu** den Favoriten hinzufügen" (the "als" is OK, though).

**Comment by ahelmberger** [comment]
This (apart from looking nice) really elevates the usefulness and clarity of the favorite tab. Cool :heart_eyes:

**Comment by adrianaluchian** [comment]
> This (apart from looking nice) really elevates the usefulness and clarity of the favorite tab. Cool 😍

Yes! And makes it more fun to use :D

File: src/components/favorites-tab.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/favorites-tab.yml]: I see 🤔 The "als" phrasing matches currently the tooltips for the favorite star "Als Favorit hinzufügen". If you think "zu den Favoriten hinzufügen" sounds better, I will change in both places.

Meantime, updated the headlines. Thanks!

**Review by ahelmberger — APPROVED** [review]

File: src/components/favorites-tab.yml (hunk 1 above)
Comment (ahelmberger) [reply src/components/favorites-tab.yml]: Cool, thanks!

**Pull request merged**